src/cluster_topology_observer.py

```diff
@@ -85,13 +86,14 @@ def start_observer(self):
                 self._charm.unit.name,
                 self._charm.charm_dir,
             ],
-            stdout=open(LOG_FILE_PATH, "a"),
+            # File shouldn't close
+            stdout=open(LOG_FILE_PATH, "a"),  # noqa: SIM115
```

Comment:
We should most probably set the output in the running instance instead or switch to a proper service.